ironfish-cli/src/commands/wallet/import.ts

```diff
@@ -43,7 +43,7 @@ export class ImportCommand extends IronfishCommand {
     }
 
     if (account === null) {
-      this.log('No account to import provided')
+      this.log('No wallet to import provided')
```

Comment:
This is incorrect: this command is used to import a single account into a wallet, which contains multiple accounts.